src/main/resources/cucumber-junit-re-runner.vm

```diff
@@ -0,0 +1,171 @@
+import cucumber.runtime.Runtime;
+import cucumber.runtime.io.MultiLoader;
+import cucumber.runtime.io.ResourceLoaderClassFinder;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.junit.Test;
+import com.github.temyers.options.ExtendedRuntimeOptions;
+import java.io.*;
+import java.util.*;
+
+
+public class $className{
+
+        private String outputPath="$outputPath";
+        private String glue =$glue;
+
+        @Test
+        public void reRun()  {
+
+                defaultRun();
+
+                int count= $retryCount;
+
+                System.out.println("Retry Count specified is :- "+count);
+
+                switch (count){
```

Comment:
Yeah, it was more around keeping it clean.

Is the output of each test run retained?

Thinking about this more, I'm actually not sure if it's a desirable feature since it is pointing to an underlying issue with the tests - they are flaky.  If you re-run an automated test without making changes, you should really get the same results.

Flaky tests are an anti-pattern:
- https://semaphoreci.com/community/tutorials/how-to-deal-with-and-eliminate-flaky-tests
- https://www.thoughtworks.com/insights/blog/no-more-flaky-tests-go-team

I'll add this comment into the main issue